packages/mongo-nav/src/mongo-select/OrgSelect.tsx

```diff
@@ -380,7 +383,13 @@ function OrgSelect({
             )}
 
             <ul className={ulStyle}>
-              {filteredData?.map(renderOrganizationOption) ?? (
+              {isAdminSearch && isFetching && (
+                <li className={emptyStateStyle}>Searching...</li>
+              )}
+              {isAdminSearch && !isFetching && renderedData.length === 0 && (
+                <li className={emptyStateStyle}>No matches found</li>
+              )}
```

Comment:
This will be revised with Nelle's styles but for now I wanted to get some visual indicator for admins that something was happening.